dev/google-drive-files/generate-file-ids.rb

```diff
@@ -19,7 +19,12 @@
 SCOPES = ["https://www.googleapis.com/auth/drive.readonly"]
 OPENARM_HARDWARE_FOLDER_ID = "1a9ec9vzBV_D-AX9s_LOkBVy3ZXDC1kJT"
 
-def list_files(drive, folder_id, parent_path: "", page_token: nil)
+def output?(name, version)
+  return true unless name =~ /_v\d+\.\d+_/
```

Comment:
Ruby is an object oriented language. So regular expression object should be a receiver. :-)

```suggestion
  return true unless /_v\d+\.\d+_/.match?(name)
```